diff --git a/node_modules/@easyops-cn/docusaurus-search-local/dist/client/client/theme/SearchBar/SearchBar.jsx b/node_modules/@easyops-cn/docusaurus-search-local/dist/client/client/theme/SearchBar/SearchBar.jsx
index 96ed430..dfdf6c7 100644
--- a/node_modules/@easyops-cn/docusaurus-search-local/dist/client/client/theme/SearchBar/SearchBar.jsx
+++ b/node_modules/@easyops-cn/docusaurus-search-local/dist/client/client/theme/SearchBar/SearchBar.jsx
@@ -30,17 +30,7 @@ async function fetchAutoCompleteJS() {
     return autoComplete;
 }
 async function fetchOpenAskAI() {
-    try {
-        const openAskAIModule = await import("open-ask-ai");
-        await import("open-ask-ai/styles.css");
-        return {
-            AskAIWidget: openAskAIModule.AskAIWidget,
-        };
-    }
-    catch (error) {
-        // open-ask-ai is optional, return null if not available
-        return null;
-    }
+  return null;
 }
 const SEARCH_PARAM_HIGHLIGHT = "_highlight";
 export default function SearchBar({ handleSearchBarToggle, }) {
